Google.PowerShell/PubSub/GcpsTopic.cs

```diff
@@ -0,0 +1,234 @@
+                        WriteObject(getRequest.Execute());
+                    }
+                    catch (GoogleApiException ex) when (ex.HttpStatusCode == HttpStatusCode.NotFound)
+                    {
+                        ErrorRecord errorRecord = new ErrorRecord(
+                            new ItemNotFoundException($"Topic '{topicName}' does not exist in project '{Project}'."),
+                            "TopicNotFound",
+                            ErrorCategory.ObjectNotFound,
+                            topicName);
+                        WriteError(errorRecord);
+                    }
+                }
+            }
+            else
+            {
+                ProjectsResource.TopicsResource.ListRequest listRequest = Service.Projects.Topics.List($"projects/{Project}");
+                do
+                {
+                    ListTopicsResponse response = listRequest.Execute();
+
+                    if (response.Topics != null)
+                    {
+                        WriteObject(response.Topics, true);
+                    }
+                    listRequest.PageToken = response.NextPageToken;
+                }
+                while (!Stopping && listRequest.PageToken != null);
+            }
+        }
+    }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Removes Google Cloud PubSub topics.
+    /// </para>
+    /// <para type="description">
+    /// Removes one or more Gooogle Cloud PubSub topics. Will throw errors if the topics do not exist.
+    /// The cmdlet will delete the topics in the default project if -Project is not used.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Remove-GcpsTopic -Topic "my-topic"</code>
+    ///   <para>This command removes topic "my-topic" in the default project.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Remove-GcpsTopic -Topic "topic1", "topic2" -Project "my-project"</code>
+    ///   <para>
+    ///   This command removes 2 topics ("topic1" and "topic2") in the project "my-project".
+    ///   </para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/pubsub/docs/publisher#delete)">
+    /// [Deleting a Topic]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "GcpsTopic")]
+    public class RemoveGcpsTopic : GcpsCmdlet
+    {
+        /// <summary>
+        /// <para type="description">
+        /// The project to remove the topics in. If not set via PowerShell parameter processing, will
+        /// default to the Cloud SDK's DefaultProject property.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [ConfigPropertyName(CloudSdkSettings.CommonProperties.Project)]
+        public string Project { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The names of the topics to be removed.
```

Comment:
Rename to `Topics`? Change text to "topic(s)"? Something just doesn't seem right about the tense here, though if I am wrong about the PowerShell idiom, do whatever is the standard.